hello-jni/app/build.gradle

```diff
@@ -2,7 +2,7 @@ apply plugin: 'com.android.application'
 
 android {
     compileSdkVersion 25
-    buildToolsVersion '25.0.2'
+    buildToolsVersion '26.0.2'
```

Comment:
Delete this line.

You no longer need to explicitly specify the build tools version. AGP uses a recommended default automatically :)

Reply:
done

Reply:
done